client/src/app/components/answer-table/answer-table.tsx

```diff
@@ -66,7 +66,13 @@ const AnswerTable: React.FC<IAnswerTableProps> = ({
           <Tr>
             <TableHeaderContentWithControls {...tableControls}>
               <Th {...getThProps({ columnKey: "choice" })} />
-              <Th {...getThProps({ columnKey: "weight" })} />
+              <Th
+                {...getThProps({ columnKey: "weight" })}
+                info={{
+                  tooltip:
+                    "This column shows the associated risk weight of an answered question. A red 'X' icon indicates High risk, a yellow warning icon indicates Medium risk, and a green checkmark indicates Low risk.",
```

Comment:
Since the `tooltip` prop can be a `ReactNode`, I'd rather see a better styled tooltip:

```jsx
tooltip: (
  <TextContent>
    <Text>This column shows the associated risk weight of an answered question.</Text>
    <List isPlain>
      <ListItem>{getIconByRisk("red")} is High risk</ListItem>
      <ListItem>{getIconByRisk("yellow")} is Medium risk</ListItem>
      <ListItem>{getIconByRisk("green")} is Low risk</ListItem>
    </List>
  </TextContent>
),
```

Doing some i18n and adding the messages to `translation.json` would be nice.

Reply:
For some reason this is the result of the tooltip with ReactNode elements
<img width="451" alt="Screenshot 2023-10-05 at 3 00 40 PM" src="https://github.com/konveyor/tackle2-ui/assets/11218376/53ff9c33-9211-4435-98d2-007cc45e4776">